libs/vm_monitor/src/cgroup.rs

```diff
@@ -53,7 +56,7 @@ pub struct CgroupWatcher {
 
 impl CgroupWatcher {
     /// Create a new `CgroupWatcher`.
-    #[tracing::instrument(skip_all, fields(%name))]
+    #[tracing::instrument(skip_all, fields(% name))]
```

Comment:
```suggestion
    #[tracing::instrument(skip_all, fields(%name))]
```

to avoid unnecessary diff and remain consistent with the codebase